src/frameworks/index.ts

```diff
@@ -513,6 +514,12 @@ ${firebaseDefaults ? `__FIREBASE_DEFAULTS__=${JSON.stringify(firebaseDefaults)}\
         // continue
       });
 
+      if (fileExistsSync(getProjectPath(".npmrc"))) {
```

Comment:
I'd rather use the async APIs so we won't have any hiccups if we want to parallelize this work later, mind changing?